### PR TITLE
Strict is deprecated

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,7 +14,7 @@ deps =
 passenv = ARCHFLAGS LDFLAGS CFLAGS INCLUDE LIB LD_LIBRARY_PATH USERNAME PYTHONIOENCODING OPENSSL_FORCE_FIPS_MODE
 commands =
     pip list
-    pytest -n auto --cov=cryptography --cov=tests --capture=no --strict --durations=10 {posargs}
+    pytest -n auto --cov=cryptography --cov=tests --capture=no --strict-markers --durations=10 {posargs}
```
Comment (alex) [inline tox.ini:17]: What version of pytest was `strict-markers` added in, we should probably set that as the minimum.

Comment (reaperhulk) [reply tox.ini:17]: Looks like this happened with 6.2, which also dropped support for Python less than 3.6. Fortunately that’s fine for us

Comment (alex) [reply tox.ini:17]: is this by way of saying you'll change our setup.py, or you don't want to because python_requires already makes thie always true?

Comment (reaperhulk) [reply tox.ini:17]: I’ll change it when I’ve got a full computer available 😀

**Review by alex — APPROVED** [review]

**Pull request merged**